src/main/scala/firrtl/ExecutionOptionsManager.scala

```diff
@@ -298,6 +299,15 @@ trait HasFirrtlOptions {
       "use this to force appending annotation file to annotations being passed in through optionsManager"
     }
 
+  parser.opt[String]("output-annotation-file")
+    .abbr("foaf")
```

Comment:
What does foaf mean?

Reply:
foaf = firrtl output annotation file, its like the previous style for shortened commandline flags